src/vm/versionresilienthashcode.cpp

```diff
@@ -26,47 +27,47 @@ int GetVersionResilientTypeHashCode(IMDInternalImport *pMDImport, mdExportedType
     LPCUTF8 szNamespace;
     LPCUTF8 szName;
     bool hasTypeToken = true;
-    int hashcode = 0;
```

Comment:
Might make sense to keep local variable `hashcode`, and use it for storing the intermediate results. Set `*pdwHashCode` just before `return true`.